wiki/Article_Styling_Criteria/en.md

```diff
@@ -101,22 +118,40 @@ Folder names should not use characters that require percent encoding. Those char
 
 The name of an article file can be found in the `File Name` column of [locales](#locales). The location of a translation is to be placed in the appropriate English-named folder.
 
-### Index pages
+### Index articles
 
-All folders must contain a page of some kind, even if they are index pages that link to their subfolders. Index pages must link to their own subfolders and could contain descriptions of what each link links to.
+All folders must contain a page of some kind, even if they are index articles that link to their subfolders. Index articles must link to their own subfolders and could contain descriptions of what each link links to.
+
+### Disambiguation articles
+
+[Disambiguation](/wiki/Disambiguation) articles must be placed in the `/wiki/Disambiguation` folder.
 
 ## Markdown
 
 These rules are to be followed for all articles.
 
 ### GFM
 
-Parts of the GFM (GitHub Flavored Markdown) markup is supported in the osu! wiki. GFM has a small amount of markup that **anyone can learn in about 3 minutes**! However, do not expect a lot out of GFM as it is intended to be used as a bare minimum markup language.
+Parts of the GFM (GitHub Flavored Markdown) markup is supported in the osu! wiki. GFM has a small amount of markup that anyone can learn in about 3 minutes. However, do not expect a lot out of GFM as it is intended to be used as a bare minimum markup language.
 
 #### HTML
 
 **HTML use is prohibited!** If you need to use any HTML for any reason, you must reconsider the layout you are using.
 
+##### HTML entities
+
+*For more information about HTML entities, see [Entity - Glossary (MDN)](https://developer.mozilla.org/en-US/docs/Glossary/Entity)*
+
+HTML character entites should almost never be used. If used however, prefer the character over the entity. For example:
+
+```
+Do not: the bestest free-to-win rhythm game&trade;
+Do not: the bestest free-to-win rhythm game&#153;
+Do not: the bestest free-to-win rhythm game&#x99;
+
+Do:    the bestest free-to-win rhythm game™
```

Comment:
Changing back the contractions ruined the spacing alignment.

Reply:
Just noticed that a few hours ago and was working on it, but then I decided to re-read the whole then again.